incubator/kubirds-powerapi-reactor/templates/reactor.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: kubirds.com/v1
+kind: Reactor
+metadata:
+  name: {{ include "kubirds-powerapi-reactor.fullname" . }}
+  labels:
+    {{- include "kubirds-powerapi-reactor.labels" . | nindent 4 }}
+    {{ if .Values.labels }}
+    {{ .Values.labels | toYaml | nindent 4 }}
+    {{ end}}
+spec:
+  unitSelector:
+    {{ if .Values.unitSelector }}
+    {{ .Values.unitSelector | toYaml | nindent 4 }}
+    {{ end}}
+  image:
+    name: powerapi/{{ .Values.kind }}-formula
```

Comment:
You forgot the tag, this will default to `latest`, but this should be configurable.